### ISSUE TITLE
print human error message for ASPA pre-16

### ISSUE THREAD

**Issue by ties** [NLnetLabs/routinator#904]
Hi,

I see the following output when encountering ASPA objects of the profile before the AS was removed,
```
[WARN] rsync://rpki.sub.apnic.net/repository/A914BC7A0000/0/AS148807.asa: missing further values (at position 2).
[WARN] rsync://rpki.sub.apnic.net/repository/A914BC7A0000/0/AS148804.asa: missing further values (at position 2).
[WARN] rsync://rpki.sub.apnic.net/repository/A914BC7A0000/0/AS148801.asa: missing further values (at position 2).
[WARN] rsync://rpki.sub.apnic.net/repository/A914BC7A0000/0/AS148802.asa: missing further values (at position 2).
[WARN] rsync://rpki.sub.apnic.net/repository/A914BC7A0000/0/AS148810.asa: missing further values (at position 2).
[WARN] rsync://rpki.sub.apnic.net/repository/A914BC7A0000/0/AS148809.asa: missing further values (at position 2).
[WARN] rsync://rpki.sub.apnic.net/repository/A914BC7A0000/0/AS148803.asa: missing further values (at position 2).
[WARN] rsync://rpki.sub.apnic.net/repository/A914BC7A0000/0/AS148806.asa: missing further values (at position 2).
[WARN] rsync://rpki.sub.apnic.net/repository/A914BC7A0000/0/AS148800.asa: missing further values (at position 2).
[WARN] rsync://rpki.sub.apnic.net/repository/A914BC7A0000/0/AS148808.asa: missing further values (at position 2).
[WARN] rsync://rpki.sub.apnic.net/repository/A914BC7A0000/0/AS148805.asa: missing further values (at position 2).
```

It might be nice to make this a human friendly message that says they likely are objects that follow a deprecated object format that is no longer accepted?

**Comment by partim** [comment]
Agreed. There’s probably more places where this sort of thing happens.

**Comment by partim** [comment]
Fixed in #917.
